docs/content/docs/deep/container-and-component-loading.md

```diff
@@ -25,225 +25,295 @@ as well as the finished diagram in [Appendix 1](#appendix-1).
 Finally, as you read through this doc you will find yourself having lots of questions. This is good, and intentional!
 Keep reading as it's likely explained later.
 
+## Pre-requisites
+
+This document assumes knowledge or familiarity with the following concepts:
+
+- What the op stream is
```

Comment:
Can we link to the appropriate docs for these?

Reply:
We should also probably use the expanded version of the term here, since it's now the first time we reference "op stream"
```suggestion
- What the Operation (op) Stream is
```